en/simulation/index.md

```diff
@@ -269,7 +269,7 @@ PX4 connects/integrates with this camera in _exactly the same way_ as it would w
 1. [TRIG_INTERFACE](../advanced_config/parameter_reference.md#TRIG_INTERFACE) must be set to `3` to configure the camera trigger driver for use with a MAVLink camera
    :::tip
    In this mode the driver just sends a [CAMERA_TRIGGER](https://mavlink.io/en/messages/common.html#CAMERA_TRIGGER) message whenever an image capture is requested.
-   For more information see [Camera](../peripherals/camera.md).
+   For more information see [Cameras Connected to Flight Controller Outputs](../camera/fc_connected_camera.md).
```

Comment:
TBD. This might need to be completely removed if we're not using the trigger interface/don't need it. I don't think we should.